benchopt/plotting/plot_objective_curve.py

```diff
@@ -70,7 +68,7 @@ def plot_objective_curve(df, plotly=False, suboptimality=False,
 
         fill_between_x(
             fig, curve['time'], q1, q9, curve['objective_value'],
-            color=CMAP(i), marker=markers[i], label=solver_name, plotly=plotly
+            color=CMAP(i), marker=MARKERS[i], label=solver_name, plotly=plotly
```

Comment:
```suggestion
            color=CMAP(i), marker=markers[i], label=solver_name, plotly=plotly
```

Reply:
and for the markers I think >52 curves should work. For the color, I guess it will cycle earlier (I have not updated that , so it should be approx. 20 as before, except if some modulo is used for cycling on the colormap?)